content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,302 @@
+to **only** show records that match all the rules contained inside. This uses the same logic as an
+AND (&) logical operator.
+
+.. _phone-number:
+
+3. Contact method
+~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule to the filter. Set the first field to
+:guilabel:`Phone`. Then, select :guilabel:`is set` from the drop-down menu in the second field.
+Selecting this operator **only** filters for records that have a phone number associated with the
+lead.
+
+Click :guilabel:`New rule` and set the first field to :guilabel:`Email`. Then, select :guilabel:`is
+set` from the drop-down menu in the second field.
+
+These rules add only leads with an associated contact method to the report.
+
+.. _active-status:
+
+1. Active status
+~~~~~~~~~~~~~~~~
+
+Click the :icon:`fa-sitemap` :guilabel:`(Branch)` icon to the right of the `Phone is set` line, to
+add a new rule that branches from the rules above.
+
+Two horizontal sets of fields appear below a line showing :guilabel:`any of:` with a
+:icon:`fa-caret-down` :guilabel:`(down caret)` icon. This setting filters for records that match
+**any** of the rules contained inside. This uses the same logic as an OR (|) logical operator.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is set` in the next field.
+
+Next, click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is set` to
+create a new line of fields beneath it.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is not set` in the next field.
+
+.. image:: lead_distribution_report/active-set.png
+  :align: center
+  :alt: Use Active to include active status in the report.
+
+This rule adds the activity status of the lead to the report.
+
+.. note::
+  Active status is an important filter to include when creating a lead distribution report because
+  it includes **all** leads regardless of won/lost or active/inactive status in the report. This
+  provides a comprehensive view of all the leads assigned to each sales member.
+
+Filter for quality leads
+------------------------
+
+The following additional conditions are provided as an example of a *good*, but *not comprehensive*,
+set of rules for finding quality leads. These filters should be applied on top of the
+:ref:`essential-filters` in the order specified to achieve a heavily-detailed filter.
+
+:ref:`referred-by`
+
+:ref:`source`
+
+:ref:`notes`
+
+:ref:`tags`
+
+:ref:`email`
+
+:ref:`salesperson`
+
+These conditions can be added, removed, or modified to best fit the desired information in the
+report.
+
+.. _referred-by:
+
+1. Referred by
+~~~~~~~~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is not set` to
+create a new line of fields beneath it.
+
+Next, set the first field to :guilabel:`Referred By`. In the second field, select
+:guilabel:`contains` from the drop-down menu. In the last field, type `appointment`.
```

Comment:
The value of `appointment` is only valuable to us because it's a string that gets passed from our internal Appointments application. It might be a different value or not a relevant rule to include (regardless of the filter's value) depending on the database. I wouldn't frame this as a concrete instruction to follow, but more as a "hey this is an example of a Referred By  rule, in this case for a business that leverages the **Appointments** app and tags those leads accordingly."